system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -65,7 +65,7 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
     const std::chrono::milliseconds measurement_period,
```

Comment:
Constructor doc needs to be updated with this parameter, can document behavior as well for the default.

Reply:
Done.